sensor_id: 0
width: 640
height: 480
fps: 10
